### ISSUE TITLE
Version not bumping

### ISSUE THREAD

**Issue by ralphsmith80** [conventional-changelog/standard-version#604]
I have a remote instance (Jenkins) running the release script. That's been working fine but running it on a new instance is producing different results. The issue is that the bump part of the release script is not occurring. The script is literally the same and I'm using `npm ci` to ensure the package versions are consistent. 

Any thought on why this could produce different results running the same script on different systems? Are there some git repository expectations that might be missing? It produces the right tag it's just not bumping the version. 

**New instance** output. Note the missing bump logs.
```
+ npm run release -- --release-as patch

> @mob/mobile@1.0.0 release /var/lib/jenkins/workspace/commonjsmodule-publish_master
> standard-version "--release-as" "patch"

✔ outputting changes to CHANGELOG.md
✔ committing CHANGELOG.md
✔ tagging release v1.0.1
ℹ Run `git push --follow-tags origin master` to publish
```

Running **locally**.
```
$ npm run release -- --release-as major

> @mob/mobile@1.0.0 release /Users/ralph.smith/Workspace/mobile
> standard-version "--release-as" "patch"

✔ bumping version in package.json from 1.0.0 to 1.0.1
✔ bumping version in package-lock.json from 1.0.0 to 1.0.1
✔ outputting changes to CHANGELOG.md
✔ committing package-lock.json and package.json and CHANGELOG.md
✔ tagging release v1.0.1
ℹ Run `git push --follow-tags origin master && npm publish` to publish
```

Here's a historical log from the **old instance** that's been working fine:
```
+ npm run release -- --release-as patch

> @mob/mobile@0.1.0 release /var/jenkins_home/workspace/_commonjsmodule-publish_master_2
> standard-version "--release-as" "patch"

✔ bumping version in package.json from 0.1.0 to 0.1.1
✔ bumping version in package-lock.json from 0.1.0 to 0.1.1
✔ outputting changes to CHANGELOG.md
✔ committing package-lock.json and package.json and CHANGELOG.md
✔ tagging release v0.1.1
ℹ Run `git push --follow-tags origin master && npm publish` to publish
```

**Comment by droshev** [comment]
hi guys, we are facing same issue. Is there a chance we can bring more attention to this issue?

**Comment by praneybehl** [comment]
Yeah facing similar issue, even for changes with 'feat' are pushed as minor.

**Comment by jbadan** [comment]
@praneybehl We found the 🐛  for our implementation at least - it ended up being something in our `.gitignore` triggering this line: https://github.com/conventional-changelog/standard-version/blob/master/lib/lifecycles/bump.js#L155

See https://github.com/SAP/fundamental-styles/pull/1289

Hope that helps you too!

**Comment by praneybehl** [comment]
@jbadan Thank you so much for the super quick reply. But mind if I ask what is the fix? As I can see the PR is already merged to master and I am on v9.0.0

**Comment by jbadan** [comment]
@praneybehl I'm not a standard-version contributor. Our repo - the closed pr I linked - had this same problem that's described in this issue - the version wasn't bumping and there was no logging output. 
The root cause ended up being some extra files listed in our gitignore that were triggering the line in standard-version that I linked. Not sure if that's your same problem but it's worth investigating 🍀

**Comment by praneybehl** [comment]
@jbadan Thank you, I will investigate the cause further. It looks like I also have a couple of similar entries

`
coverage
node_modules
npm-debug.log*
.cache

# Project #
storybook-static
dist
es
lib
ts
package-lock.json
.eslintcache
*.swp
jest-test-results.json
__coverage__
.out
src/**/*.d.ts

# Packages #
# it's better to unpack these files and commit the raw source
# git has its own built in compression methods
*.7z
*.dmg
*.gz
*.iso
*.jar
*.rar
*.tar
*.zip

# Text editor files #
.idea
.vscode

# Logs #
*.log

# OS generated files #
.DS_Store?
._*
.Spotlight-V100
.Trashes
ehthumbs.db
Thumbs.db


/utils/
/theme/
/Button/
`

**Comment by dawidchyrzynski** [comment]
#751

**Comment by mapperCoderZ** [comment]
Wow you saved me, because our gitlab repo has been moved to a gitlab group "/dist/", standard-version was not bumping anymore its versions, it's crazy. Fixed the .gitignore line "dist" to "/dist".
